app/views/comments/_comment.html.erb

```diff
@@ -11,11 +11,14 @@
     </div>
     <div>
       <% if current_user == comment.author %>
-        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted", remote: true do %>
```

Comment:
Good start. Try adding a unique dom_id to your button. This makes the element being manipulated more specific